includes/Classifai/Command/ClassifaiCommand.php

```diff
@@ -280,6 +280,109 @@ public function image( $args = [], $opts = [] ) {
 		}
 	}
 
+	/**
+	 * Batch crop image(s).
+	 *
+	 * ## Options
+	 *
+	 * [<attachment_ids>]
+	 * : Comma delimeted Attachment IDs to crop.
+	 *
+	 * [--limit=<limit>]
+	 * : Limit classification to N attachments. Default 100.
+	 *
+	 * [--skip=<skip>]
+	 * : Skip first N attachments. Default false.
+	 *
+	 * @param array $args Arguments.
+	 * @param array $opts Options.
+	 */
+	public function crop( $args = [], $opts = [] ) {
+		$classifier     = new ComputerVision( false );
+		$settings       = $classifier->get_settings();
+		$smart_cropping = new SmartCropping( $settings );
+
+		if ( ! isset( $settings['enable_smart_cropping'] ) || '1' !== $settings['enable_smart_cropping'] ) {
```

Comment:
Just my own opinion here and would like further input from others, but should we worry about checking the setting before running? Seems like if someone is manually running this WP-CLI command, they don't care if the feature is enabled in the admin or not, they want it to run regardless.

Reply:
I can see where they don't want someone running this from WP Admin but are ok running it via CLI, so I'm ok with not checking the setting though open to alternatives as well?